2_LSA/README.md

```diff
@@ -1,5 +1,27 @@
 ## LSA - Light Sensor Array
 
+### Table of Contents
+* [Overview](#overview)
+* [Components of LSA](#components-of-lsa)
+* [Function of Components](#function-of-components)
+	* [LED](#function-of-components)
+	* [PhotoDiode](#function-of-components)
+	* [IC LM324](#function-of-components)
+	* [Resistors](#function-of-components)
+* [Working Principle](#working-principle)
+* [Appplications](#applications)
+* [Scope of Improvement](#scope-of-improvement)
+* [Description of Functions Used](#description-of-functions-used)
+
+### Overview 
+- The Light Sensor Array(LSA) is used to detect the white/black lined surface and it sends this data to the microcontroller ESP32. Thus, the purpose of line following is achieved.
+
+- Physical PCB
+<img src="https://drive.google.com/uc?export=view&id=1cdTDn6es5xVDZxhYENmADuIx9tZpwBtm" width="1000" height="600">
```

Comment:
This image isn't loading, please don't use google drive links anywhere. Upload it in assets folder in your subfolder.

Reply:
Added assets folder, fixed README.md